apps/pie-storybook/.storybook/preview.js

```diff
@@ -16,4 +16,32 @@ export default {
             },
         },
     },
+    parameters: {
+      a11y: {
+        config: {
+          rules: [
+            {
+              id: 'WCAG Rules',
+              tags: [
+                  'wcag21a',
+                  'wcag21aa',
+                  'wcag143',
+                  'cat.color',
+                  'cat.aria'
+              ]
+            },
+            {
+              // Disabled rule
+              id: 'color-contrast-enhanced',
+              enabled: false,
+            },
+            {
+              // Disabled rule
+              id: 'color-contrast',
+              enabled: false,
+            },
```

Comment:
mind sharing why we are disabling them?

Reply:
So the reason we've disabled this is due to the color contrast for some of our oranges across design that causes issues. So we've disabled it in areas. Designers are aware of this issue but it would almost mean changing all Jet Orange to cater for this particular issue.